### PR TITLE
switch2containers: remove ceph packages

### PR DESCRIPTION
**Pull request by guits**
Once the migration to containers is done we can make the playbook remove
any ceph rpms if the user set `remove_pkg_post_container_migration` to
`true`

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1976986

### CODE REVIEW

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

File: infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml [hunk 1]
```diff
@@ -679,6 +679,52 @@
...
+    - "{{ nfs_group_name|default('nfss') }}"
+    - "{{ client_group_name|default('clients') }}"
+    - "{{ mgr_group_name|default('mgrs') }}"
+    - "{{ monitoring_group_name|default('monitoring') }}"
+    - "{{ iscsi_gw_group_name|default('iscsigws') }}"
+  gather_facts: false # Already gathered previously
+  tasks:
+    - name: remove ceph packages
+      package:
+        name: "{{ item }}"
```
Comment (dsavineau) [inline infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml:724]:
```suggestion
        name: "{{ ceph_packages }}"
```

**Review by dsavineau — CHANGES_REQUESTED** [review]

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Review by dsavineau — APPROVED** [review]

File: infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml [hunk 2]
```diff
@@ -679,6 +679,54 @@
...
+    - "{{ osd_group_name|default('osds') }}"
+    - "{{ mds_group_name|default('mdss') }}"
+    - "{{ rgw_group_name|default('rgws') }}"
+    - "{{ rbdmirror_group_name|default('rbdmirrors') }}"
+    - "{{ nfs_group_name|default('nfss') }}"
+    - "{{ client_group_name|default('clients') }}"
+    - "{{ mgr_group_name|default('mgrs') }}"
+    - "{{ monitoring_group_name|default('monitoring') }}"
+    - "{{ iscsi_gw_group_name|default('iscsigws') }}"
+  gather_facts: false # Already gathered previously
```
Comment (dsavineau) [inline infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml:722]:
```suggestion
  gather_facts: false # Already gathered previously
  become: true
```

**Review by dsavineau — CHANGES_REQUESTED** [review]

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Pull request closed without merging**